TESTS/mbedtls/multi/main.cpp

```diff
@@ -23,6 +23,17 @@
 
 #include "mbedtls/sha256.h"
 
+#if defined(MBEDTLS_PLATFORM_C)
+#include "mbedtls/platform.h"
+#else
+#include <stdio.h>
```

Comment:
Is there a reason to include this stuff in the ```#else``` block.

Reply:
To be consistent with other examples, and to allow compilation on platforms without PLATFORM_C define.

Reply:
@hanno-arm made a more specific comment https://github.com/ARMmbed/mbed-os/pull/6749#pullrequestreview-118312774